enhancements/two-nodes-openshift/2no.md

```diff
@@ -212,9 +252,38 @@ More information on creating OCF agents can be found in the upstream [developer
 
 Tools for extracting support information (must-gather tarballs) will be updated to gather relevant logs for triaging issues.
 
-As part of the fencing setup, the cri-o and kubelet services will still be owned by systemd when running under pacemaker. The main difference is that the resource agent will be responsible for signaling systemd to change their active states. The etcd pods are different in this respect since they will be restarted using Podman. It may be possible to start these with the same user account as the original pods.
+As part of the fencing setup, the cri-o and kubelet services will still be owned by systemd when running under pacemaker. The main difference is that the resource agent will be responsible for signaling systemd to change their active states.
+The etcd pods are different in this respect since they will be restarted using Podman, but this will be running as root, as it was under CEO.
```

Comment:
only the etcd _container_ is running under podman. I wonder, would it still require root? I would assume so, because the /var/lib/etcd folder would otherwise need a chmod

Reply:
The note about running as root was to help address a concern that the process updates changing which user runs the process could have implications like thread limitations.